src/libraries/pkg/test/packageTest.targets

```diff
@@ -136,7 +137,8 @@
 
   <Target Name="VerifyNotDependsOnNetStandard"
           DependsOnTargets="ResolveReferences"
-          Condition="'$(_ShortFrameworkIdentifier)' == 'net' AND '$(_TargetFrameworkVersionWithoutV)' &gt;= '4.6.1' AND '$(DisableVerifyNotDependsOnNetStandardTest)' != 'true'">
+          Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' and
+                      '$(DisableVerifyNotDependsOnNetStandardTest)' != 'true'">
```

Comment:
@joperezr just FYI, `VerifyNotDependsOnNetStandard` wasn't running for the last year as the internal properties that it depended on were removed in the SDK.